### PR TITLE
Order layers

### PR DESCRIPTION
**Pull request by eranra**

### CODE REVIEW

File: src/Config.ts [hunk 1]
```diff
@@ -18,23 +18,31 @@
...
-const WEIGHT_CONTAINERS = 20
-const WEIGHT_K8S_FEDERATION = 100
-const WEIGHT_K8S_CLUSTER = 101
-const WEIGHT_K8S_NODE = 102
-const WEIGHT_K8S_NAMESPACE = 103
-const WEIGHT_K8S_POD = 104
-const WEIGHT_K8S_CONTAINER = 105
-const WEIGHT_K8S_SERVICE = 106
-const WEIGHT_K8S_OTHER = 120
+const SHOW_DEBUG = "True"
```
Comment (safchain) [inline src/Config.ts:21]: nit: just SHOW_DEBUG = true

And I would prefer to have it set to false as default

File: src/Config.ts [hunk 2]
```diff
@@ -541,106 +558,111 @@ class DefaultConfig {
                 attrs.weight = WEIGHT_K8S_OTHER
         }
 
+        if (SHOW_DEBUG.toLowerCase() == "true") {
```
Comment (safchain) [inline src/Config.ts:561]: just  `if (SHOW_DEBUG) {`

File: src/Config.ts (hunk 1 above)
Comment (eranra) [reply src/Config.ts]: done

File: src/Config.ts (hunk 2 above)
Comment (eranra) [reply src/Config.ts]: done

**Pull request merged**